integration/middleware-test.ts

```diff
@@ -1074,7 +1253,7 @@ test.describe("Middleware", () => {
       await page.waitForSelector("[data-child]");
 
       // 2 separate server requests made
-      expect(requests).toEqual([
+      expect(requests.sort()).toEqual([
```

Comment:
This test was flaky in Firefox due to the order of requests not being deterministic.